packages/graphql/tests/integration/custom-resolvers-int.test.ts

```diff
@@ -236,7 +235,7 @@ describe("Custom Resolvers", () => {
                         typeDefs = `
                             type Query {
                                 test: ${type}! @cypher(statement: """
-                                RETURN \"${id}\"
+                                RETURN \\"${id}\\"
```

Comment:
Out of interest, why is the extra escaping necessary here? Are we passing this complexity onto users?

Reply:
Now we are putting the given cypher query inside either apoc.runFirstColumn or apoc.doIt and so, with your highlighted, the first braces are to escape GraphQL and the second is to escape inside Cypher. 

```
CALL apoc.doIt("RETURN \"some-id\"")
```